src/omnisharpServerLauncher.ts

```diff
@@ -20,20 +21,34 @@ export interface LaunchResult {
     command: string;
 }
 
-export function installOmnisharpIfNeeded(): Promise<string> {
+export function installOmnisharpIfNeeded(output: OutputChannel): Promise<string> {
     return getOmnisharpLaunchFilePath().catch(err => {
+        if (getSupportedPlatform() == SupportedPlatform.None && process.platform === 'linux') {
+            output.appendLine("[ERROR] Could not locate an OmniSharp server that supports your Linux distribution.");
+            output.appendLine("");
+            output.appendLine("OmniSharp provides a richer C# editing experience, with features like IntelliSense and Find All References.");
+            output.appendLine("It is recommend that you download the version of OmniSharp that runs on Mono using the following steps:");
+            output.appendLine("    1. If it's not already installed, download and install Mono (http://www.mono-project.com)");
+            output.appendLine("    2. Download and untar https://github.com/OmniSharp/omnisharp-roslyn/releases/download/v1.9-alpha13/omnisharp-linux-mono.tar.gz");
```

Comment:
I'll file an issue to make this a bit more robust for RTM. We don't want to have to update this URL everytime we release.